v2/types.go

```diff
@@ -27,6 +27,11 @@ type Service struct {
 	// Bindable represents whether a service is bindable.  May be overridden
 	// on a per-plan basis by the Plan.Bindable field.
 	Bindable bool `json:"bindable"`
+	// BindingRetrievable represents whether fetching a service binding via
+	// a GET on the binding resource's endpoint
+	// (/v2/service_instances/instance-id/service_bindings/binding-id) is
+	// supported for all plans.
+	BindingRetrievable bool `json:"binding_retrievable,omitempty"`
```

Comment:
This needs to be designated as an alpha field and noted that it will only be set if the client has enabled alpha features

Reply:
Done.